### ISSUE TITLE
Github folder name incorrectly named

### ISSUE THREAD

**Issue by JohnReddyZhang** [uchicago-mobi/MPCS51030-2019-Winter-Forum#140]
Hi, I have a Models folder in my local repository, but on Github it is somehow named to "New Group", how can I change the folder name on GitHub?
I have tried changing the name locally and pushing it, this did not work.
Thank you!

![屏幕快照 2019-03-12 下午7 33 04](https://user-images.githubusercontent.com/15935574/54245226-018e5d80-44fe-11e9-93fb-ea64b1039423.png)
<img width="228" alt="屏幕快照 2019-03-12 下午7 34 59" src="https://user-images.githubusercontent.com/15935574/54245227-018e5d80-44fe-11e9-8f5b-303920719622.png">

**Comment by susanstevens** [comment]
You could try seeing what the folder is called in Finder. It used to be that the project navigation structure in Xcode didn't necessarily match what was in Finder, although it seems like they usually match by default now. Maybe this particular group/folder isn't matching, though.

**Comment by JohnReddyZhang** [comment]
I see, the folder in finder is indeed called "New Group". Should I change the folder name? It seems that after changing the name Xcode will lose sync with the files inside.

**Comment by susanstevens** [comment]
Yes, after you change the folder name in Finder you would need to drag the files from Finder back into Xcode. As long as they aren't displaying in red after that, you should be good. You can always double-check that everything is there by pushing your changes and cloning a new copy of your repo.

**Comment by JohnReddyZhang** [comment]
I tried changing the folder name and by just dragging them back did not really make any difference. I think what need to be done is to change the folder's reference so project structure also changes (Clicking on the group - on the right change its "location" property under file inspector.
